src/main/java/ch/njol/skript/conditions/CondIsBlockRedstonePowered.java

```diff
@@ -25,25 +25,40 @@
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
 
 @Name("Is Block Redstone Powered")
-@Description("Checks if a block is powered by redstone")
+@Description("Checks if a block is (indirectly) powered by redstone")
```

Comment:
I don't think ``(indirectly)`` is suitable here, it should just be ``Checks if a block is indirectly or directly powered by redstone``